hoot-core/src/main/cpp/hoot/core/info/ConfigOptionsDisplayer.cpp

```diff
@@ -34,11 +34,25 @@
 
 // Qt
 #include <QFile>
+#include <QProcess>
+#include <QTextCodec>
+#include <QTextStream>
 
 namespace hoot
 {
 
-void ConfigOptionsDisplayer::displayAllOptionNames(const bool withDetails)
+QString runProcess(const QString& cmd)
+{
+  QProcess proc;
+  proc.start("bash", QStringList() << "-c" << cmd);
+  if (!proc.waitForStarted())
+    throw HootException("Unable to list configuration options.");
+  if (!proc.waitForFinished())
+    throw HootException("Unable to list configuration options.");
+  return QString(proc.readAll());
+}
```

Comment:
What does this do?

Reply:
The system call outputs the results to the display (as part of the `cout` removal) this captures the output as a string and returns it.

Reply:
I'll have to take a look around for more system calls and see if they output to the terminal too.